Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'puppet-lint-classes_and_types_beginning_with_digits-check'
 gem 'puppet-lint-empty_string-check'
 gem 'puppet-lint-file_ensure-check'
 gem 'puppet-lint-leading_zero-check'
+gem 'puppet-lint-legacy_facts-check'
```

Comment:
In general I consider it bad practice to do that in a PR where you're also doing something else. At the very least it needs to be a separate commit but please keep PRs to a single topic. If it's something you really depend on, it's common practice to send a preparation PR. The reason is that we also generate a changelog based on this and dropping legacy facts means also dropping support for Facter 2. In itself that's not a problem, but something that should be in the changelog (and is possibly a reason for a major version bump).